src/main/java/access2csv/Driver.java

```diff
@@ -129,25 +140,44 @@ public static void main(String[] args) throws Exception {
 			parser.printHelpOn(System.out);
 			return;
 		}
-
+      
 		File inputFile = input.value(options);
 		if(!inputFile.exists()) {
 			throw new FileNotFoundException("Could not find input file: " + inputFile.toString());
 		}
 		
-		File outputDir = output.value(options);
-		if(!outputDir.exists()) {
-			outputDir.mkdirs();
+		File outputDir = null;
+		if (options.has(output)) {
+			outputDir = output.value(options);
+			if(!outputDir.exists()) {
+				outputDir.mkdirs();
+			}	
 		}
 		
-		if(options.has(schema)) {
-			exportAll(inputFile, options.has(withHeader), outputDir, csvPrefix.value(options));
+		boolean applyQuotesToAll = true;
+			if (options.has(quoteAll)) {
```

Comment:
do we need the `has` check? I think the default value you specified above lets us unconditionally use `boolen applyQuotesToAll(quoteAll.value(options))`

Reply:
I'll look at it again.

Reply:
I think you're right about this. I'll make the change and test it out.